client/webserver/site/src/js/markets.js

```diff
@@ -812,13 +812,12 @@ export default class MarketsPage extends BasePage {
     if (order.targetID) {
       const targetOrder = this.metaOrders[order.targetID]
       if (!targetOrder) return
-      Doc.hide(Doc.tmplElement(targetOrder.row, 'cancelStatus'))
       if (note.subject === 'cancel') {
-        targetOrder.order.status = Order.StatusCanceled
+        Doc.tmplElement(targetOrder.row, 'cancelStatus').textContent = 'canceled'
```

Comment:
Once canceled, we can just hide the `cancelStatus`, since it will say so in the order status column. The `cancelStatus` is just to indicate that a cancellation is pending, since the order status column must still say that the order is booked. 

In reality, we could probably simplify here. Since this was written, I've added the `statusString` method to `orderutil.js`, which is supposed to indicate 'cancelling'. Not certain yet why it's not doing that for the order status column already.

Do you mind if I take a look at this separately?

Reply:
Please do, thanks!  Closing.